dockerize.go

```diff
@@ -18,32 +19,22 @@ import (
 type sliceVar []string
 type hostFlagsVar []string
 
-type Context struct {
-}
-
-func (c *Context) Env() map[string]string {
-	env := make(map[string]string)
-	for _, i := range os.Environ() {
-		sep := strings.Index(i, "=")
-		env[i[0:sep]] = i[sep+1:]
-	}
-	return env
-}
-
```

Comment:
I found this confusing -- I thought it was somehow related to golang.org/x/net/context.Context and the ctx  variable.  So I renamed it to EnvContext and moved it into templates.go, which is the only place its used